app/assets/javascripts/school_administrator_dashboard/DashboardScatterPlot.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import HighchartsWrapper from '../components/HighchartsWrapper';
+import  hash from 'object-hash';
+
+// Component for all charts in the dashboard page.
+export default class DashboardScatterPlot extends React.Component{
+
+//highcharts has trouble combining zoom with a render, so rendering is prevented unless the displayed data changes
+  shouldComponentUpdate(nextProps) {
```

Comment:
@kevinrobinson I didn't (just) put this here to bug you. Highcharts seems to have trouble when the selection function is a callback that sets state. Preventing an unnecessary render resolves the issue.